confignode/src/main/java/org/apache/iotdb/confignode/service/thrift/server/ConfigNodeRPCServerProcessor.java

```diff
@@ -186,14 +187,15 @@ public TDataPartitionResp getOrCreateDataPartition(TDataPartitionReq req) throws
 
   @Override
   public TSStatus operatePermission(TAuthorizerReq req) throws TException {
-    if (req.getAuthorType() < 0 || req.getAuthorType() >= PhysicalPlanType.values().length) {
-      throw new IndexOutOfBoundsException("Invalid ordinal");
+    if (req.getAuthorType() < 0
+        || req.getAuthorType() >= AuthorOperator.AuthorType.values().length) {
+      throw new IndexOutOfBoundsException("Invalid Author Type ordinal");
     }
     AuthorPlan plan = null;
     try {
       plan =
           new AuthorPlan(
-              PhysicalPlanType.values()[req.getAuthorType()],
+              AuthorPlan.convert(req.getAuthorType()),
```

Comment:
Could you please explain the change?

Reply:
The authorization operation type passed from the client is the index of the passed AuthorType. If the PhysicalPlanType is used here to obtain the corresponding type or cause the wrong order